src/main/java/ua/vedroid/cinema/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,70 @@
+package ua.vedroid.cinema.dao.impl;
+
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import ua.vedroid.cinema.dao.OrderDao;
+import ua.vedroid.cinema.exception.DataProcessingException;
+import ua.vedroid.cinema.lib.Dao;
+import ua.vedroid.cinema.model.Order;
+import ua.vedroid.cinema.model.User;
+import ua.vedroid.cinema.util.HibernateUtil;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    private static final Logger log = LogManager.getLogger(OrderDaoImpl.class);
+
+    @Override
+    public Order add(Order order) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(order);
+            transaction.commit();
+            log.info("Added new order: " + order);
+            return order;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can`t insert order entity " + order, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getAll() {
```

Comment:
Do we need this method?

Reply:
if we want to get a list of all orders?
+ consistency, all DAOs have this method